src/test/java/org/openstreetmap/atlas/geography/atlas/statistics/AtlasStatisticsTest.java

```diff
@@ -214,6 +215,78 @@ public void testCountingWater()
                 statistics.get(new StatisticKey("", "pier_surface", "true")).getCount(), 0.01);
     }
 
+    @Test
+    public void testCountingAois()
```

Comment:
It would be nice to have an Assert statement per new count line. But given the size of that update, I understand if you feel this is enough.